velox/functions/prestosql/tests/BitwiseTest.cpp

```diff
@@ -120,6 +126,17 @@ TEST_F(BitwiseTest, bitwiseAnd) {
   EXPECT_EQ(bitwiseAnd<int64_t>(kMin64, -1), kMin64);
 }
 
+TEST_F(BitwiseTest, bitCount) {
+  EXPECT_EQ(bitCount<int64_t>(9, kMaxBits), 2);
```

Comment:
i) Add more negative test cases for 'bits' argument...near edge values
ii) Try negative values for it.

Reply:
Done.